backend/app/models/project.py

```
@@ -53,6 +53,9 @@ class Project(ProjectBase, table=True):
     collections: list["Collection"] = Relationship(
         back_populates="project", cascade_delete=True
     )
+    fine_tuning: list["Fine_Tuning"] = Relationship(
```

Ruff (0.12.2)

Undefined name `Fine_Tuning`

(F821)